core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -76,6 +79,12 @@ class AdminClient(val time: Time,
     response.groups().asScala.map(group => GroupOverview(group.groupId(), group.protocolType())).toList
   }
 
+  def getApiVersions(node: Node): util.Collection[ApiVersion] = {
```

Comment:
I think we should return a Scala collection like other methods (e.g. `listGroups`).